static/js/browser2/arc_section.tsx

```diff
@@ -96,24 +95,13 @@ export class ArcSection extends React.Component<
   }
 
   private fetchData(): void {
-    const labelsPromise = axios
+    axios
       .get("/api/browser/proplabels/" + this.props.dcid)
-      .then((resp) => resp.data);
-    const provenancePromise = axios
-      .get("/api/browser/triples/Provenance")
-      .then((resp) => resp.data);
-    Promise.all([labelsPromise, provenancePromise])
-      .then(([labelsData, provenanceData]) => {
-        const provDomain = {};
-        for (const prov of provenanceData) {
-          if (prov["predicate"] === "typeOf" && !!prov["subjectName"]) {
-            provDomain[prov["subjectId"]] = new URL(prov["subjectName"]).host;
-          }
-        }
+      .then((resp) => {
+        const labelsData = resp.data;
```

Comment:
So now each page only needs to do one API call to fetch the provenance domain?

Reply:
We originally only needed to do one API call to fetch the provenance domain for the arcs. But now that we also need the provenance domain for weather charts, I moved fetching the provenance domain to a parent component so that we still only fetch the provenance domain once.